src/util.ts

```diff
@@ -85,3 +85,14 @@ export const exec = async (
       }
     }
   );
+
+export const throwAssertNever = (value: never) => {
```

Comment:
small nit but I don't think this is a significant advantage over `throw assertNever(value)` we save on typing whitespace and this potentially messes with the stack trace since the throw would come from this file and not the original location.

Reply:
The issue I had is that `throw assertNever(value)` doesn't compile in a ternary expression:
![image](https://github.com/p0-security/p0cli/assets/5836460/d8d31aaf-3970-49d0-a810-8fbe188f31f2)

Reply:
I see, TIL!